Kickstarter-iOS/Views/Storyboards/SettingsNotifications.storyboard

```diff
@@ -26,30 +26,27 @@
                                 <color key="backgroundColor" cocoaTouchSystemColor="groupTableViewBackgroundColor"/>
                             </tableView>
                             <pickerView contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="9gJ-Cv-1eW">
-                                <rect key="frame" x="0.0" y="1000" width="375" height="200"/>
+                                <rect key="frame" x="0.0" y="1000" width="375" height="216"/>
                                 <color key="backgroundColor" red="0.90980392160000001" green="0.90980392160000001" blue="0.90980392160000001" alpha="1" colorSpace="calibratedRGB"/>
-                                <constraints>
```

Comment:
Good job! Don't know why we initially set the height to 200 but 216 looks closer to native `UIPicker` dimensions.